bedrock/settings/base.py

```diff
@@ -406,6 +406,19 @@ def lazy_langs():
 SESSION_COOKIE_SECURE = not DEBUG
 SESSION_ENGINE = 'django.contrib.sessions.backends.cache'
 
+# legacy setting. backward compat.
+DISABLE_SSL = config('DISABLE_SSL', default=True, cast=bool)
+# SecurityMiddleware settings
+SECURE_HSTS_SECONDS = config('SECURE_HSTS_SECONDS', default='0', cast=int)
+SECURE_HSTS_INCLUDE_SUBDOMAINS = False
+SECURE_BROWSER_XSS_FILTER = config('SECURE_BROWSER_XSS_FILTER', default=False, cast=bool)
+SECURE_SSL_REDIRECT = config('SECURE_SSL_REDIRECT', default=not DISABLE_SSL, cast=bool)
```

Comment:
As long as we're discussing variable names, `SECURE_SSL_REDIRECT` is both redundant and inaccurate. How about shortening it to just `SECURE_REDIRECT`? If we wanted to get rid of `DISABLE_SSL` at the same time, we could change this to 
```python
SECURE_REDIRECT = config('SECURE_REDIRECT', default=True, cast=bool)
```
This wouldn't require any changes to deployed Deis apps, but it would mean changes to local dev .env files, and updating a couple of other places where we currently refer to that variable. I'm fine with leaving `DISABLE_SSL` in place for now for backwards compat and addressing that in a separate PR though.